.github/workflows/git-actions.yml

```diff
@@ -25,7 +25,8 @@ jobs:
           flake8 . --count --show-source --statistics
       - name: Run unit tests
         run: |
+          python3 -c "from pyesbulk import __VERSION__; print(__VERSION__)"
```

Comment:
Note in the Travis/GitHub-Action logs that _this_ command works just fine; however in the `pytest` run, none of the test modules can find `pyesbulk`.

`pytest` runs just fine locally, and I'm having trouble figuring out what could be happening here.

Reply:
That is because it seems `python3` adds the current directory by default when the interpreter is invoked directly, but doesn't add the path when invoked via the file header command method.